test/workerd/tests.mjs

```diff
@@ -36,14 +36,12 @@ export const url_parse = {
 export const buffer_implements = {
   async test() {
     const encoder = new TextEncoder();
-    const buffer = await import("unenv/runtime/node/buffer");
+    const buffer = await import("node:buffer");
```

Comment:
@pi0 How do you propose covering `nodeless` support for this change? Should this line be reverted?

Reply:
Maybe having two test exports in the meantime?

(if too hard please keep the version you prefer for now)

Reply:
I can update the PR.